Formula/lmdb.rb

```diff
@@ -16,20 +15,15 @@ class Lmdb < Formula
   end
 
   def install
-    inreplace "libraries/liblmdb/Makefile" do |s|
-      s.gsub! ".so", ".dylib"
-      s.gsub! "$(DESTDIR)$(prefix)/man/man1", "$(DESTDIR)$(prefix)/share/man/man1"
+    cd "libraries/liblmdb" do
+      inreplace "Makefile", ".so", ".dylib"
```

Comment:
Not entirely, but eh, passing `"SOEXT=.dylib"` to `make` in future isn't the worst thing in the world.